DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml

```diff
@@ -22,7 +22,7 @@
             <a asp-action="@nameof(DFC.ServiceTaxonomy.GraphVisualiser.Controllers.VisualiseController.Viewer)"
                asp-controller="Visualise"
                asp-area="@typeof(DFC.ServiceTaxonomy.GraphVisualiser.Startup)!.Namespace"
-               asp-route-visualise="@graphSyncPartText"
+               asp-route-visualise="@(graphSyncPartText +"&graph=draft")"
```

Comment:
Is this the best way of doing this?

Reply:
(the graph name)

Reply:
I'd probably inject `GraphSyncHelper` and the 2 `ContentItemVersion`s into the view and call `GetIdPropertyValue()` to get `graphSyncPartText`.